packages/theming/android-theme/src/androidTypography.ts

```diff
@@ -17,33 +17,35 @@ export function androidTypography(): Typography {
       heroLarge: 28 as FontSize, // Headline
     } as FontSizes,
     weights: {
-      light: '300' as FontWeightValue,
+      // The font weights do not work for Android since RN 0.60 (https://github.com/facebook/react-native/issues/25696)
+      // The workaround is to use font aliases from android source (https://android.googlesource.com/platform/frameworks/base/+/master/data/fonts/fonts.xml)
       regular: '400' as FontWeightValue,
-      medium: '500' as FontWeightValue,
       semiBold: '500' as FontWeightValue,
     },
     families: {
-      primary: 'Roboto',
+      primary: 'sans-serif', // sans-serif, weight=400
+      primaryBold: 'sans-serif-medium', // sans-serif, weight=500
```

Comment:
Should this be named primaryMedium instead? primaryBold makes me think the font weight would be '700' instead of '500'.

Reply:
Yep! changed to primarySemibold in the updated change